app/helpers/recipes_helper.rb

```diff
@@ -0,0 +1,21 @@
+module RecipesHelper
+
+  def preparation_time_hours(minutes)
+    min = (minutes % 60).to_s + "min"
+    hours = (minutes/60).floor().to_s + "h "
```

Comment:
brackets `()` after `.floor()` can be skipped :)

also I don't see `@preparation_time_hours` used anywhere in the code. is this variable needed? if no you can just left `minutes>59 ? hours + min : min` at the end of the method.

the same question goes to other methods in this file